dts/arm/st/stm32l4.dtsi

```diff
@@ -170,6 +170,31 @@
 			status = "disabled";
 			label = "SPI_2";
 		};
+
+		usbotg_fs: otgfs@50000000 {
+			compatible = "st,stm32-otgfs";
+			reg = <0x50000000 0x40000>;
+			interrupts = <67 0>;
+			interrupt-names = "otgfs";
+			num-bidir-endpoints = <1>;
+			num-in-endpoints = <5>;
+			num-out-endpoints = <5>;
+			ram-size = <1280>;
+			status = "disabled";
+			label= "OTGFS";
+		};
+
+		can1: can@40006400 {
+			compatible = "st,stm32-can";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40006400 0x400>;
+			interrupts = <19 0>, <20 0>, <21 0>, <22 0>;
+			interrupt-names = "TX", "RX0", "RX1", "SCE";
```

Comment:
If interrupt-names is required , you should add it in yaml binding (as optional since f0 do not use it).
It seems it worked even though it should be present in your case, so generation script would require a fix (or not).
Anyway, please fix the binding.

Reply:
Done. The unused bindings of port and pin will be removed in the rework PR